src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (<p className="Person__age">{`I am ${age}`}</p>) : ''}
```

Comment:
bad practice, Misha said about it in one of pre-tasks videos :)

rather use:
`   {age ? (
<p className="Person__age">
{`I am ${age}`}
</p>) : ''}`